### PR TITLE
XRT-2213 Function added to create binary data from string data

### PR DESCRIPTION
**Pull request by SteveOss**

### CODE REVIEW

**Comment by SteveOss** [comment]
recheck

**Comment by SteveOss** [comment]
Had to fix Sonar Qube warning (not a code issue).

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -349,3 +349,4 @@
 - Added YAML reading function `iot_data_from_yaml`
 - Added component stopping callback support (called by container before components stopped)
 - Support removed for Alpine 3.15 and Ubuntu 18.04
+- Added function `iot_data_binary_from_string` to convert binray data to string data
```
Comment (BinduRao2018) [inline RELEASE_NOTES.md]: Typo - Binary
Is the function to convert binary from/to string data - function name and description doesn't match

**Review by BinduRao2018 — COMMENTED** [review]
Description in Release Notes to be updated

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -349,3 +349,4 @@
 - Added YAML reading function `iot_data_from_yaml`
 - Added component stopping callback support (called by container before components stopped)
 - Support removed for Alpine 3.15 and Ubuntu 18.04
+- Added function `iot_data_binary_from_string` to convert binary data to string data
```
[the remark this answers is not in the document]
Comment (SteveOss) [reply RELEASE_NOTES.md]: Should be fixed

**Comment by BinduRao2018** [comment]
@SteveOss , Good to go, I can't merge as I don't have the permission.

**Pull request merged**